datareservoirio/client.py

```diff
@@ -26,6 +26,8 @@
 _END_DEFAULT = 9214646400000000000  # 2262-01-01
 _START_DEFAULT = -9214560000000000000  # 1678-01-01
 
+_TIMEOUT_DEAULT = 120
```

Comment:
I would rather have this to be 30 seconds with increasing backoff?

Reply:
We had bad experience with 4insight servers before... how about starting here, then tightening it after a while?

Reply:
That is fine for me.